projects/plugins/boost/app/assets/src/js/features/error-notice/error-notice.tsx

```diff
@@ -27,8 +29,12 @@ const ErrorNotice = ( {
 	const description = standardizeError( error ).message;
 
 	return (
-		<div className={ styles[ 'error-notice' ] }>
-			<div>
+		<div
+			className={ classNames( styles[ 'error-notice' ], {
+				[ styles[ 'variant-module' ] ]: variant === 'module',
+			} ) }
+		>
```

Comment:
After last night's conversation, I'm going to agree to this because it changes the internal layout of the component - parents shouldn't be aware of their child's layouts.